src/services/notificationHandler.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const NotificationHandler = require('../lib/notificationHandlerWebsocket')
```

Comment:
That's a fair reason for changing it from worker. I'm not sure about handler though. In js event "handlers" are listeners..